src/hooks/mutations/useGetAllUsers.jsx

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from "@tanstack/react-query";
+import axios from "axios";
+
+
+const NewRequestTable = async () => {
```

Comment:
Please update the name of this function that reflects what it's doing. The name NewRequestTable does not reflect you retrieving users